docs/dev/w3c-webdriver-capabilities.md

```diff
@@ -155,35 +155,62 @@ Browser Options classes are used to manage both browser specific functionality a
 
```

Comment:
@diemol about the Verifying Your Tests for Compliance section (line 134), was thinking of replacing it with: "To verify W3C compliance, take a look at the driver response of your session request. If it contains _only_ W3C valid caps, it is a W3C session. WebdriverIO can sends both types of capabilities -- WebDriver and JWP; if you see them both, your session is out of compliance."

Can you let me know what you think?

Reply:
I think "compliance" sounds kind of old school, don't you think? Maybe we can say "compatibility" instead?

I believe that checking the session response might be a good idea. However, it is more about the response shape. I also think that checking the session response might be complicated because you need to be able to inspect it, and in some languages is not straightforward. What if we suggest our customers to check the "metadata" tab in the test results page?

E.g.
![image](https://user-images.githubusercontent.com/5992658/143430858-45d07ebb-a87a-4560-9fef-9bf5061d52ff.png)

Reply:
That sounds good to me